From 2bf858a8c92aadaf75ce8213ea037fe7db544ae8 Mon Sep 17 00:00:00 2001
From: Pino Toscano <toscano.pino@tiscali.it>
Date: Sat, 2 Jan 2021 14:26:52 +0100
Subject: [PATCH] build: stop using -Werror

-Werror turns every single compiler warning into an error; while this
seems nice to catch potential issues, in reality:
- newer versions of Qt deprecated more APIs (and currently some APIs
  deprecated in Qt 5.14 & 5.15 are still used)
- newer versions of the compiler can be more strict, and generate more
  warnings; sometimes they are not really issues

Hence, stop using -Werror altogether.
---
 common-project-config.pri | 2 --
 1 file changed, 2 deletions(-)

diff --git a/common-project-config.pri b/common-project-config.pri
index c0d2e80..a2ffac0 100644
--- a/common-project-config.pri
+++ b/common-project-config.pri
@@ -19,8 +19,6 @@ UI_SOURCES_DIR  = ui/src
 
 QMAKE_CXXFLAGS += -fno-exceptions \
     -fno-rtti
-# we don't like warnings...
-unix:QMAKE_CXXFLAGS += -Werror
 
 TOP_SRC_DIR     = $$PWD
 
-- 
2.29.2

